app/services/booking_response.rb

```diff
@@ -1,9 +1,44 @@
 class BookingResponse
-  def initialize(success:)
-    @success = success
+  ALREADY_PROCESSED_ERROR = 'already_processed'.freeze
+  PROCESS_REQUIRED_ERROR = 'process_required'.freeze
+  NOMIS_VALIDATION_ERROR = 'nomis_validation_error'.freeze
+  NOMIS_API_ERROR = 'nomis_api_error'.freeze
+
+  attr_reader :error
+
+  def self.succesful
+    new(nil)
+  end
+
+  def self.process_required
+    new(PROCESS_REQUIRED_ERROR)
+  end
+
+  def self.nomis_validation_error
+    new(NOMIS_VALIDATION_ERROR)
```

Comment:
these to `NOMIS_VALIDATION_ERROR` and `NOMIS_API_ERROR` should really be in the Nomis API service class instead of being bundled with other unrelated error messages